setup.cfg

```diff
@@ -11,44 +11,23 @@ classifier =
     Topic :: Internet :: WWW/HTTP :: Dynamic Content
     Intended Audience :: Developers
     Programming Language :: Python
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3.5
```

Comment:
Remove?

Reply:
Have follow up PR to remove this file completely